surfsense_backend/app/connectors/discord_connector.py

```
@@ -0,0 +1,264 @@
+        self.token = token
+        logger.info("Token set successfully. You can now start the bot with start_bot().")
+    
+    async def _wait_until_ready(self):
+        """Helper to wait until the bot is connected and ready."""
+        logger.info("Waiting for the bot to be ready...")
+
+        # Give the event loop a chance to switch to the bot's startup task.
+        # This allows self.start() to begin initializing the client.
+        # Terrible solution, but necessary to avoid blocking the event loop.
+        await asyncio.sleep(1) # Yield control to the event loop
+        
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)